### PR TITLE
feat: improvements on glossary Asynchoronous

### PR DESCRIPTION
**Pull request by PassionPenguin**
### Description

- separate asynchronous into two sub-glossary entries
- improvements on description on both entries.

### CODE REVIEW

**Comment by Josh-Cena** [comment]
@PassionPenguin In these PRs I see a lot of text being rewritten but I fail to see a clear quality improvement. Granted, the new content is correct and well-explained, but I don't think the previous content was inferior in any way. I wonder what the motivation is.

**Comment by hamishwillee** [comment]
Sorry to jump in @Josh-Cena, but I agree with you. There is some improvement, but a lot of "churn". This matters because on balance I'm not sure the improvement is worth the reviewer effort that it costs.

MDN actually has a stated policy that we don't accept grammar-only fixes unless they address a clear ambiguity or technical issue. @PassionPenguin Do you think you might be able to instead concentrate on things that are actually known technical issues? Such as the issues that are marked as "Good First Issues"? I don't want to sound unappreciative, because it is clear you're helping - perhaps just not in the ways that are best value for effort.

**Comment by PassionPenguin** [comment]
> Sorry to jump in @Josh-Cena, but I agree with you. There is some improvement, but a lot of "churn". This matters because on balance I'm not sure the improvement is worth the reviewer effort that it costs.
> 
> 
> 
> MDN actually has a stated policy that we don't accept grammar-only fixes unless they address a clear ambiguity or technical issue. @PassionPenguin Do you think you might be able to instead concentrate on things that are actually known technical issues? Such as the issues that are marked as "Good First Issues"? I don't want to sound unappreciative, because it is clear you're helping - perhaps just not in the ways that are best value for effort.

thats a good point. thx for your clear suggestion

**Comment by PassionPenguin** [comment]
> @PassionPenguin In these PRs I see a lot of text being rewritten but I fail to see a clear quality improvement. Granted, the new content is correct and well-explained, but I don't think the previous content was inferior in any way. I wonder what the motivation is.

just try to possibly do a content expansion or update some outdated content. sorry for bothering you

**Comment by hamishwillee** [comment]
Removing my assignment - I don't have time to review in a timely manner.

File: files/en-us/glossary/asynchronous/networking/index.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: Asynchronous (Networking)
+slug: Glossary/Asynchronous/Networking
+page-type: glossary-definition
+---
+
+{{GlossarySidebar}}
+
+In networking and other communications, **asynchronous communication** refers to the transmission of data in which each character or unit of data is sent separately, and the timing between these characters or data units does _not_ need to be {{glossary("synchronous", "synchronized")}}. This means that the sender and receiver do not need to be perfectly aligned in time, allowing data to be sent intermittently rather than in a continuous stream. Each data unit is typically framed with start and stop bits to signify the beginning and end of transmission, ensuring that the receiving end can correctly identify and process the data.
```
Comment (Josh-Cena) [inline files/en-us/glossary/asynchronous/networking/index.md:9]: This page significantly deviates from the original document, and it's unclear what you are trying to fix. I like the original wording much better and I find this a bit unpenetrable.

**Comment by hamishwillee** [comment]
OK, that was my first impression. I think splitting the doc up could be useful, but this isn't broken so is not a priority. Closing.

**Pull request closed without merging**